### PR TITLE
Tests for chat/views except failed_author_is_receiver in get and post…

### PR DESCRIPTION
**Pull request by ashatna**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVPYTP-130)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/tests/unittests/chat/testview.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        """
+        data = {'users': [100, 200, 300]}
+        users = CustomUser.objects.all()
+        for user in users:  # set user id in redis db
+            REDIS_HELPER.set(user.id, user.email)
+        data_users_online = {str(user.id): user.email for user in users}
+        url = reverse('online_status')
+        response = self.client.post(url, json.dumps(data), content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+        self.assertJSONEqual(response.content.decode('utf-8'), json.dumps(data_users_online))
```
Comment (JjuliaP) [inline eventually/tests/unittests/chat/testview.py:206]: Add blank line to the end of file.

File: eventually/tests/unittests/chat/testview.py [hunk 2]
```diff
@@ -0,0 +1,208 @@
+"""
+Chat Views Test.
+==================
+
+This module provides complete testing for all Chat's views functions.
+"""
+import datetime
+import json
+from unittest import mock
+
```
Comment (redmyt) [inline eventually/tests/unittests/chat/testview.py]: do we need this new line ?

File: eventually/tests/unittests/chat/testview.py [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 404)
+
+    def test_get_failed_author_is_interlocutor(self):
+        """
+        Method that tests the unsuccessful get request for creating a page with chat messages.
+        Author is interlocutor.
+        """
+        url = reverse('pages', args=[1500, 1])
+        c = Client()
```
Comment (redmyt) [inline eventually/tests/unittests/chat/testview.py]: please use meaningful names for the variables

File: eventually/tests/unittests/chat/testview.py [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+        self.assertEqual(response.status_code, 400)
+
+    def test_post_failed_author_is_receiver(self):
+        """
+        Method that tests the unsuccessful post request for chat message.
+        Author is receiver.
+        """
+        data = {"text": 'some_text'}
+        url = reverse('chat', args=[1500])
+        c = Client()
```
Comment (redmyt) [inline eventually/tests/unittests/chat/testview.py]: please use meaningful names for the variables

File: eventually/tests/unittests/chat/testview.py (hunk 1 above)
Comment (ashatna) [reply eventually/tests/unittests/chat/testview.py:206]: done!

File: eventually/tests/unittests/chat/testview.py (hunk 2 above)
Comment (ashatna) [reply eventually/tests/unittests/chat/testview.py]: done!

**Review by lhalam — APPROVED** [review]

**Pull request merged**